http_calls.py

```diff
@@ -446,10 +455,7 @@ def __init__(self, url, args=None):
             body = body[g_array_field]
 
         global lastResponse
-        print(lastResponse.info())
-
-        self.result = {'status_code': lastResponse.getcode(), 'headers': lastResponse.info().dict, 'body': body}
-
+        self.result = {'status_code': lastResponse.getcode(), 'headers': lastResponse.info(), 'body': body}
```

Comment:
Test: LockingApi.LeaseCredentials:
http.client.HTTPMessage in "info()" does not have dict method